platform/web/detect.py

```diff
@@ -52,6 +52,11 @@ def get_opts():
             "Use Emscripten PROXY_TO_PTHREAD option to run the main application code to a separate thread",
             False,
         ),
+        BoolVariable(
+            "editor_ensure_crossorigin_isolation_headers",
```

Comment:
Same discussion as https://github.com/godotengine/godot/pull/86089#discussion_r1425677281, I think that it's more common for a hyphenated word to be snaked cased with an underscore than without.

So all `crossorigin` / `CROSSORIGIN` should be `cross_origin` / `CROSS_ORIGIN`, which is way more readable IMO.

Reply:
Though I see we already merged stuff without the underscore so I guess I won't die on that hill and let's keep it consistent...